exchange/howto/manual-dispute-payout.adoc

```diff
@@ -0,0 +1,184 @@
+
+=== Step 2. Get arbitrator's public and private key
+
+*Remove your wallet password* (by following the same steps as found in <<remove-password>> in the trader instructions section).
+
+*Open the Wallet Data* dialog with `cmd-j` and *check the `Include private keys` checkbox*.
+
+*Copy the contents of the Wallet Data dialog* and paste it into a text editor.
+
+Note the `ARBITRATOR` address at the top of the file in the BTC Wallet Address entry list, i.e. the entry with `address=1DBX...` below. This address should be the same as the one that you receive funds to as an arbitrator:
+
+    BTC Wallet:
+    Address entry list:
+    AddressEntry{offerId='null', context=ARBITRATOR, address=1DBX...}
+    AddressEntry{offerId='null', context=AVAILABLE, address=1HN2...}
+
+Now search the file for the first match for the string `addr:1DBX...`. You should find a section that looks like the following:
+
+    Key to watch:  xpub...
+     addr:1ABC...  hash160:f778...
+     DeterministicKey{pub HEX=02ff..., priv HEX=dfc8..., ...}
+     addr:1DBX...  hash160:760d...
+     DeterministicKey{pub HEX=039d..., priv HEX=929d..., ...}
+
+*Note both the `pub HEX` and `priv HEX` values for the `addr:1DBX...` entry*, which should be the second entry in the list.
+
+You can now *reset your wallet password*.
+
+=== Step 3. Get trader's private key
+
+First, navigate back to the trade dispute ticket in question. Be sure to click on the *winner's* dispute ticket (as determined in the first step of these instructions) and click `cmd-U` to re-open the ticket.
+
+Send the trader their public key (as determined in the steps above) and a link to the [trader instructions] section of this document below. Ask them to follow those instructions and to respond to you with their private key value once they have it.
+
+When you have the trader's private key, move on to the next step.
+
+=== Step 4. Issue the manual dispute payout
+
+Now that you have all the necessary information, you can *open the Emergency multi-sig payout tool* by clicking `alt-g` in the dispute view:
+
+image:images/multisig-payout-tool.png[Emergency multi-sig payout tool]
+
+NOTE: Several values are pre-filled in the payout tool. They are a left-over from testing in an earlier version of Bisq. *You should erase these pre-filled values and replace them with the ones you have collected in the steps above*.
+
+depositTxHex:: The Deposit transaction ID from the Dispute details view, i.e. the same transaction ID you clicked on to view the transaction in a block explorer.
+
+buyerPayoutAmount:: The amount in BTC that the buyer should be paid out, as noted above, e.g. `0.03`
+
+sellerPayoutAmount:: The amount in BTC the seller should be paid out, e.g. `0.0662`
+
+arbitratorPayoutAmount:: This value should always be `0`, as we no longer issue payouts to arbitrators
+
+Tx fee:: The BTC value of the `txFee` in Satoshis as taken from the JSON contract details above, e.g. `0.00036`
```

Comment:
Suggest: 
The BTC value of the `txFee` in Bitcoin as taken from the JSON contract details above (convert Satoshi value to Bitcoin), e.g. `0.00036`